build-output-api/isr-revalidation/README.md

```diff
@@ -0,0 +1,17 @@
+# On-demand Incremental Static Regeneration (ISR) Revalidation
+
+## Build Output API
```

Comment:
I suggest renaming the example folder to `on-demand-isr-revalidation`.

Reply:
Sounds good. Moved!